scipy-1.0/paper.tex

```diff
@@ -1065,6 +1024,47 @@ \subsubsection*{Test suite}
 
 % NOTE: is there a citation for PyPy?
 
+\subsubsection*{Benchmark suite}
+
+The airspeed velocity (\texttt{asv}\cite{asvref}) library enables benchmarking Python packages
+over their lifetimes, and the performance of the SciPy code base was monitored
+with \texttt{asv} starting in February of 2015 (PR \#4501). In addition to ensuring that
+unit tests are passing (see Test suite section below), confirming 
```

Comment:
The test suite section isn't below anymore so the parenthetical note can be removed. We could probably just not mention the unit test suite in the benchmark section rather than moving the benchmark section, but I don't really care if you prefer this ordering.

Reply:
I did forget to remove the parenthetical note. Fixed.